### PR TITLE
fix `setup.py`

### PR DESCRIPTION
**Pull request by Borda**
fix missing arguments anduse Utils to validate workflows and is the package can be installed

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,4 +21,6 @@
         "bitsandbytes",
     ],
     packages=find_packages(),
+    long_description=readme,
```
Comment (williamFalcon) [inline setup.py:24]: negative
@lantiga

Comment (williamFalcon) [reply setup.py:24]: don't bloat this codebase please...
let's keep all this engineering stuff out (@lantiga )

Comment (Borda) [reply setup.py:24]: ok, closing it; just be aware that the setup is not valid :)
also, let's state in the contribution section that we do not expect any code health contributions...

**Review by lantiga — APPROVED** [review]
Thanks for fixing setup.py, approved

**Pull request merged**